data/json/items/tool_armor.json

```
@@ -392,6 +392,285 @@
+  },
+  {
+    "id": "ar_glasses_t1_on",
+    "copy-from": "ar_glasses_t1",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "pair of AR glasses (on)", "str_pl": "pairs of AR glasses (on)" },
+    "description": "This pair of AR glasses is turned on and playing some great tunes, raising your morale steadily while on your person.  It runs through batteries quickly; you can turn it off by using it.  It also obscures your hearing.",
+    "power_draw": 1000,
+    "revert_to": "ar_glasses_t1",
+    "use_action": [ "MP3_ON" ],
+    "flags": [ "ONLY_ONE", "WATCH", "ALARMCLOCK", "OUTER", "FRAGILE", "TRADER_AVOID" ]
+  },
+  {
+    "id": "ar_glasses_t2",
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "name": { "str": "pair of camping AR glasses", "str_pl": "pairs of camping AR glasses" },
+    "description": "A model of augmented reality glasses adapted for camping, with adaptive light correction and some gadgets.  An augmented reality overlay allow the user to zoom, and it displays handy information about local topology.",
+    "weight": "150 g",
+    "volume": "300 ml",
+    "price": 60000,
+    "price_postapoc": 400,
+    "to_hit": -2,
+    "material": [ "glass", "plastic" ],
+    "symbol": "[",
+    "looks_like": "glasses_safety",
+    "color": "light_gray",
+    "covers": [ "eyes" ],
+    "coverage": 95,
+    "encumbrance": 6,
+    "warmth": 6,
+    "ammo": "battery",
+    "charges_per_use": 1,
+    "use_action": [ "CAMERA", "MP3" ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "holster": true,
+        "rigid": true,
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "item_restriction": [ "light_battery_cell", "light_atomic_battery_cell", "light_disposable_cell" ]
+      }
+    ],
+    "material_thickness": 1,
+    "environmental_protection": 1,
+    "flags": [ "ONLY_ONE", "ZOOM", "WATCH", "ALARMCLOCK", "OUTER", "FRAGILE", "SUN_GLASSES", "THERMOMETER", "HYGROMETER" ]
+  },
+  {
+    "id": "ar_glasses_t2_on",
+    "copy-from": "ar_glasses_t2",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "pair of camping AR glasses (on)", "str_pl": "pairs of camping AR glasses (on)" },
+    "description": "This pair of camping AR glasses is turned on and playing some great tunes, raising your morale steadily while on your person.  It runs through batteries quickly; you can turn it off by using it.  It also obscures your hearing.",
+    "power_draw": 1000,
+    "revert_to": "ar_glasses_t2",
+    "use_action": [ "MP3_ON" ],
+    "flags": [
+      "ONLY_ONE",
+      "ZOOM",
+      "WATCH",
+      "ALARMCLOCK",
+      "OUTER",
+      "FRAGILE",
+      "SUN_GLASSES",
+      "THERMOMETER",
+      "HYGROMETER",
+      "TRADER_AVOID"
+    ]
+  },
+  {
+    "id": "ar_glasses_t2_mil",
+    "type": "TOOL_ARMOR",
+    "category": "clothing",
+    "name": { "str": "military AR headset", "str_pl": "military AR headsets" },
```

```suggestion
    "name": { "str": "military AR headset" },
```
Trailing `s` will be added automatically.